vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/conversion.go

```diff
@@ -20,15 +20,14 @@ import (
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 )
 
 // AsDefaultVersionedOrOriginal returns the object as a Go object in the external form if possible (matching the
 // group version kind of the mapping if provided, a best guess based on serialization if not provided, or obj if it cannot be converted.
 // TODO update call sites to specify the scheme they want on their builder.
 func AsDefaultVersionedOrOriginal(obj runtime.Object, mapping *meta.RESTMapping) runtime.Object {
-	converter := runtime.ObjectConvertor(legacyscheme.Scheme)
-	groupVersioner := runtime.GroupVersioner(schema.GroupVersions(legacyscheme.Scheme.PrioritizedVersionsAllGroups()))
+	converter := runtime.ObjectConvertor(DefaultPrintingScheme)
```

Comment:
Hmmm... I wonder, maybe that should be configurable and not hard wired through a carry patch?

Reply:
> patch?

No.  Every usage of this function should be removed.